ignite/utils.py

```diff
@@ -56,7 +56,7 @@ def apply_to_type(
     """
     if isinstance(x, input_type):
         return func(x)
-    if isinstance(x, (str, bytes)):
+    if isinstance(x, (str, bytes, float, int)):
```

Comment:
I'm not sure about this change. I'd prefer to keep it as it was unless this is a hard requirement.